src/keystore/InviteStore.ts

```diff
@@ -4,7 +4,7 @@ import type { TopicData } from './interfaces'
 import { Mutex } from 'async-mutex'
 import { typeSafeTopicMap } from './utils'
 
-const INVITE_KEY = 'invitations:v1'
+const INVITE_KEY = 'invitations/v1'
```

Comment:
This changes seems a bit unrelated, but shouldn't have an impact right? No internal keystore state convo state is persisted yet.

Reply:
That's right. Just wanted to make the suffix consistent with the prefix (switched to slashes to make a potential `FileSystemPersistence` module prettier)